pyproject.toml

```diff
@@ -64,9 +63,8 @@ wandb = {version= "^0.12.0", optional = true}
 tqdm = {version = "^4.62.0", optional = true}
 
 [tool.poetry.extras]
-all = ["hydra-core", "neoconfigen", "pandas", "pytorch-lightning", "wandb"]
+all = ["hydra-core", "neoconfigen", "pandas", "wandb", "torch", "numpy"]
```

Comment:
pretty sure `torch` depends on `numpy`

Reply:
It does but there's no harm in being explicit